app/assets/javascripts/visualizations/highvis/highmodifiers.coffee

```diff
@@ -77,6 +79,22 @@ $ ->
     data.DEFAULT_PRECISION = 4
     data.precision ?= data.DEFAULT_PRECISION
 
+    # This should not be here.... It is also in scatter.coffee... i'll find a place for it some other time
+    globals.getCurrentPeriod = (date) =>
```

Comment:
This is your function you used to separate the data for the timeline. I used it to make the groups. Whatever value this returns I used as the group name.